### ISSUE TITLE
Bug: fa-microscope not displayed in Mozilla Firefox when not using CDN

### ISSUE THREAD

**Issue by AppLEaDaY** [FortAwesome/Font-Awesome#19947]
### Bug description

I'm using version 5.5.0 free. A list of used icons follows.

- fa-address-card
- fa-microscope
- fa-truck
- fa-chalkboard-teacher

As long as I use the CDN all the mentioned icons are displayed correctly with Mozilla Firefox.
I report the URL for the CDN below.

https://use.fontawesome.com/releases/v5.5.0/css/all.css

When I try to use a local copy of the library, though, the microscope is not displayed - the other three icons are, instead.

The URL for the zip file I downloaded follows.

https://use.fontawesome.com/releases/v5.5.0/fontawesome-free-5.5.0-web.zip

I also tried with version 6.4.0 specifying "fa-solid fa-microscope", instead of "fas fa-microscope", but with no luck.

**I have not the mentioned problem with Chromium Browser and with Chrome, I have it with Mozilla Firefox**.

I don't know how to make the proper choice in the "Implementation" selection box.

### Screenshots

- Using CDN
 
![image](https://github.com/FortAwesome/Font-Awesome/assets/10222899/b1399b3d-410e-46b0-9a34-ef7649e80f24)

- With the library installed locally with the CMS
 
![image](https://github.com/FortAwesome/Font-Awesome/assets/10222899/76ed942c-b077-4fb4-a96e-67fa315575c5)

### Font Awesome version

v5.4.0, v6.4.0

### Serving

Self-hosted

### Implementation

Other (as specified in the bug description)

### Browser and Operating System

Mozilla Firefox 118.0.1 on Ubuntu 22.04.3 LTS

### Web bug report checklist

- [ ] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
Ciao!

Thanks for being part of the Font Awesome Community.

> When I try to use a local copy of the library, though, the microscope is not displayed - the other three icons are, instead.

For some reason, maybe the font is not referenced correctly in the local environment and/or there's an older version installed in the system (I guess < 5.2.0) used as a fallback

Here it is a working reduced test case: https://tagliala.github.io/fa19947.html

Please double check that:

1. Your browser development console is loading the correct webfont file
2. There are no cache issues
3. both `webfont` and `css` folders are up to date

Closing here, but feel free to comment

**Comment by AppLEaDaY** [comment]
It's embarrassing: today I tried again (in an _private window_) and the problem has gone; this is driving me nuts, since I'm afraid by now I've no means to reproduce what happened and yesterday I spent quite a lot of time on that.
All I can see now is the microscope is displayed, the downloaded css refers actually to version 5.5.0 and a couple of woff2 files are downloaded, fa-solid-900.woff2 and fa-brands-400.woff2. I cannot tell if both the files are required in order to display the mentioned icon.

Thanks anyway!


Andrea
